test/lib/usd/translators/CMakeLists.txt

```diff
@@ -72,6 +71,15 @@ set(TEST_SCRIPT_FILES
     testUsdMayaAdaptorUndoRedo.py
 )
 
+if(BUILD_PXR_PLUGIN)
+    # This test uses the file "UsdExportUVTransforms.ma" which
+    # requires the plugin "pxrUsdPreviewSurface" that is built by the
+    # Pixar plugin.
+    list(APPEND TEST_SCRIPT_FILES
+        testUsdExportUVTransforms.py
+    )
+endif()
```

Comment:
I had built locally only the maya-usd plugin and got this test failing. Found it is because of the ma file it loads (there is another example of this).

Reply:
A better approach would be to port this test to use usdPreviewSurface node that is independent of Pixar plugin - see https://github.com/Autodesk/maya-usd/blob/dev/lib/usd/pxrUsdPreviewSurface/usdPreviewSurfacePlugin.cpp

Reply:
Let's do that in a separate PR. I copied this way of skipping the test from another place. So there are multiple tests that need to be ported.